P/polyhedral/build_tarballs.jl

```diff
@@ -29,21 +29,26 @@ export EIGEN_PATH=$includedir/eigen3
 export NAUTY_INCLUDE="-I$includedir/nauty"
 export NAUTY_LINK="-L$libdir -lnauty"
 
+export GLPK_LINK="-L$libdir -lglpk"
+
 make
```

Comment:
Maybe
```suggestion
make -j${nproc}
```
will make this less slow.